### DISCUSSION TITLE
Connectivity Issues after upgrade tasmota 12.5 to 13.x  only on esp32 devices

### DISCUSSION THREAD

**Discussion by kroon040** [arendst/Tasmota#19267] [Q&A]
Hi ,

I have 20+ devices  (esp8266, esp32- solo (shellys) , and es32 dual core (sonoff dual R3)  and running tasmota. After upgrading form 12.5 i notice often that connectivity is lost for couple seconds. After some research it is only on the esp32 devices. Also if i swicth from Homeassistant a couple of times (5-10), then also i lost connectivity. Also ping the device , the ping is lost for about 5-10 seconds. 

I have checked about 10 devices and the all have it. I have different wifi aps tested and same response. The esp8266 don't have the issue on tasmota 13.x.  Downgrading to 12.5 solves the issue

Anybody have the same issue, and anybody knows what could be the issue?

**Comment by tsmoneta** [comment]
I have over 100 devices running Tasmota 13, only 3 with esp32. Ping tests onn the esp32 show no lost pings.

200 packets transmitted, 200 received, 0% packet loss, time 199113ms
rtt min/avg/max/mdev = 12.914/87.831/599.801/50.504 ms

**Comment by kroon040** [comment]
If you switch the esp32 devices with mqtt about 5-10 times in short time , does the ping go well and keeps the devices responsive?

**Comment by tsmoneta** [comment]
OK, that symptom I've seen before - on an ESP8266 device. If you have a relay, the current surge from actuating the coil can cause a voltage drop. Have you checked to see if the ESP32 is rebooting? If so, placing a 220uF (or larger) electrolytic capacitor across the GND and 3.3V should stop it.

**Comment by kroon040** [comment]
The devices are not rebooting. I'm using shelly plus devices and also sonoff dual R3, and don't going modify it. Now I flashed the devices back to 12.5 and solves also the issue.
